core/lib/contract_verifier/src/etherscan/metrics.rs

```diff
@@ -0,0 +1,59 @@
+//! Reports etherscan verifier metrics
+use std::time::Duration;
+
+use tokio::sync::watch;
+use zksync_dal::{ConnectionPool, Core, CoreDal};
+
+use crate::metrics::CONTRACT_VERIFIER_METRICS;
+
+pub struct EtherscanVerifierMetrics {
```

Comment:
Nit: Please always derive `Debug` for your structures, as it can simplify debugging in the future (most common case: when your structure is included in some tester object, and lack of `Debug` derive forbids deriving `Debug` there).